### DISCUSSION TITLE
Add more features in events that triggers workflow

### DISCUSSION THREAD

**Discussion by beingminimal** [community/community#49376] [Actions]
### Select Topic Area

Product Feedback

### Body

Hi Community!

Github workflow is a great feature and cherry on cake is [events that triggers workflows](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows).

But, recently we found many events still not supported in this list. Like following

Team Related events like team created, updated, deleted
team member added, updated, removed
dependabot_alert
project v2 items (columns) create, edit, reorder, restore, delete
project v2 all events
repository events like created, updated, public, deleted, edited, transfer, archived, etc
[commit comment](https://docs.github.com/webhooks-and-events/webhooks/webhook-events-and-payloads#commit_comment)
Organisation Management (like rename, created, updated, deleted)
Organisation Member Added, updated, removed
Secret scanning alert
Security advisory Management
We suggest all supported events in [webhook-events-and-payloads](https://docs.github.com/en/webhooks-and-events/webhooks/webhook-events-and-payloads) should be supported in [Events that trigger workflows](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows).
Because it removes the dependency of external web hook management to create automation in github.
This will help more developers to build automation in github very easily.

We would be happy to collaborate if required from our side.
Awaiting community support. Thanks

**Comment by pfhuillet** [comment]
it should be useful to have this triggers :

- project v2 items (columns) create, edit, reorder, restore, delete
- project v2 all events
